forms-flow-api/src/formsflow_api/resources/user.py

```diff
@@ -254,3 +268,32 @@ def delete(user_id, group_id):
                 "message": "Invalid request data",
             }, HTTPStatus.BAD_REQUEST
         return None, HTTPStatus.NO_CONTENT
+
+
+@cors_preflight("POST, OPTIONS")
+@API.route(
+    "/add-tenant-user",
```

Comment:
Can we change it a generic endpoint by removing tenant reference ?

Reply:
done